ONSdigital/dp-frontend-filter-dataset-controller
# Feature/replace filter client

### What

Filter client updated to use the new RCHTTP lib

### How to review

* Check out commit
* Check the code
* Run the thing

### Who can review

Anyone except me
